component_registry_bindings/session.py

```diff
@@ -223,6 +229,21 @@ def delete(self, id):
 
     # Extra operations
 
+    def count(self, *args, **kwargs):
```

Comment:
does this take into account if the passed **params includes an include_fields param ?

Reply:
Yes, it does, however the `include_fields` param is replaced with the nonexisting field, so minimum data is transferred.